spec/rubocop/cop/betterment/active_job_performable_spec.rb

```diff
@@ -2,65 +2,92 @@
 
 describe RuboCop::Cop::Betterment::ActiveJobPerformable, :config do
   it 'rejects a performable that is a plain ruby class' do
-    inspect_source(<<-DEF)
+    inspect_source(<<~RUBY)
```

Comment:
I tried to convert these to use `expect_offense` to improve the usefulness the spec's failure message, but Rubocop tried to parse the URL in the offense and choked.